examples/Remarketing/AddRemarketingAction.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V2.Common;
+using Google.Ads.GoogleAds.V2.Errors;
+using Google.Ads.GoogleAds.V2.Resources;
+using Google.Ads.GoogleAds.V2.Services;
+
+using System;
+using System.Linq;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example adds a new remarketing action to the customer and then retrieves its
+    /// associated tag snippets.
+    /// </summary>
+    public class AddRemarketingAction : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddRemarketingAction codeExample = new AddRemarketingAction();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds a new remarketing action to the customer and then" +
+                    " retrieves its associated tag snippets.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for the conversion action is
+        /// added.</param>
+        public void Run(GoogleAdsClient client, long customerId)
+        {
+            // Get the RemarketingActionService.
+            RemarketingActionServiceClient remarketingActionService =
+                client.GetService(Services.V2.RemarketingActionService);
+
+            // Get the RemarketingActionService.
```

Comment:
RemarketingActionService to GoogleAdsService in the comment.

Reply:
Done.